README.md

```diff
@@ -1,4 +1,152 @@
+
+All tasks related to the `Django` backend are run from the project root (where this `README.md` is located).
+
+The easiest way to start working on the project is to use our `Makefile`:
+
+    $ make bootstrap
+
+This command builds the `app` container, installs back-end dependencies and performs database migrations. It's a good idea to use this command each time you are pulling code from the project repository to avoid dependency-related or migration-related issues.
+
+🔧 **Before you go further**, you should take a look at the newly created `./env.d/development` file and replace the relevant values with your own. You can take a look at the [environment documentation](https://github.com/openfun/marsha/blob/write-the-readme/docs/env.md#1-django-backend-environment) for more details on this topic.
+
+Now that your `Docker` services are ready to be used, start the application by running:
+
+    $ make run
+
+You should be able to view the development view at [localhost:8060/development/](http://localhost:8060/development/).
+
+### The storage & transcoding environment
+
+All tasks related to this environment are run from the `./src/aws` directory. We use `Terraform` to keep this infrastructure configuration as code and easily manage several independent deployments of the whole `AWS` infrastructure.
+
+🔧 **Before you go further**, you need to create `./src/aws/env.d/development` and replace the relevant values with your own. You can take a look at the [environment documentation](https://github.com/openfun/marsha/blob/master/docs/env.md#2-environment-to-deploy-marsha-to-aws) for more details on this topic. You can use this command to create the file from the existing model:
+
+    $ cp ./src/aws/env.d/development.dist ./src/aws/env.d/development
+
+Create the shared state bucket where `Terraform` will keep all the information on your deployments so different developers/machines/CI processes can interact with them:
+
+    $ make state-create
+
+Initialize your `Terraform` config:
+
+    $ make init
+
+Build the lambdas (using `yarn`) and automatically configure the infrastructure (this will start incurring billing on `AWS`):
+
+    $ make deploy
+
+Everything should be set up! You can check on your `AWS` management console.
+
+You may have noticed that the `AWS` development environment requires a URL where the `Django` backend is running. You can easily get a URL that points to your locally running `Django` app using a tool such as `ngrok`.
+
+### The `React` frontend
+
+All tasks related to the `React` frontend are run from the `./src/frontend` directory.
+
+We use `yarn` for all those tasks. Make sure you have a recent version installed:
+
+```bash
+$ yarn --version
+  1.13.0
+```
+
+If you need to install `yarn`, please take a look at [the official documentation](https://yarnpkg.com/en/docs/install).
+
+Install all the dependencies:
+
+    $ yarn install
+
+Run the build and copy the `iframe-resizer` host-side JS into your local `Django` assets:
```

Comment:
:thinking:  it does not seem related to the command below.

Reply:
`build-dev` is just the `build` + `copy-iframe-resizer`

Reply:
So maybe either:
- split in two commands or
- don't mention about the iframe-resizer if we can consider that it is part of the build...